plugins/generator-1.15.2/forge-1.15.2/templates/tool.java.ftl

```diff
@@ -342,6 +342,35 @@ public class ${name}Item extends ${JavaModName}Elements.ModElement{
 			return ${data.enchantability};
 		}
 
+	}
+<#elseif data.toolType == "Fishing rod">
+    private static class ItemToolCustom extends FishingRodItem {
+
+		protected ItemToolCustom() {
+			super(new Item.Properties().group(${data.creativeTab}).maxDamage(${data.usageCount}));
+		}
+
+		@Override public boolean hitEntity(ItemStack stack, LivingEntity target, LivingEntity attacker) {

```

Comment:
I think you can drop this one in both versions too and is probably leftover from multitool/special?